drizzlepac/haputils/product.py

```diff
@@ -288,13 +288,21 @@ def align_to_gaia(self, catalog_list=[], output=True, process_type='SVM',
                                 alignment_pars['determine_fit_quality']['do_consistency_check'] = False
 
                                 # Evaluate the quality of the fit
+                                # Need to create this log file specifically for the PyTest testing environment
+                                # when a single Python session is running and the tests are looking for dataset-specific
+                                # log files.
+                                log_file = "temp_align.log"
                                 is_good_fit, _, _, _, _, _ = align.determine_fit_quality_mvm_interface(align_table.imglist,
                                                                                                        align_table.filtered_table,
                                                                                                        more_catalogs,
                                                                                                        num_cat,
                                                                                                        alignment_pars,
                                                                                                        print_fit_parameters=True,
-                                                                                                       loglevel=self.log_level)
+                                                                                                       loglevel=self.log_level,
+                                                                                                       runfile=log_file)
+                                
+                                # Clean up the temporary log file as the contents are captured
+                                os.remove(log_file)
```

Comment:
Be careful with the use of 'os.remove' as this can fail under Windows but work perfectly well for the same data under Linux or Mac.  As a result, you may want to trap the OSError exception specifically here and report it separately from other possible reasons for failure at this point.

Reply:
Modified to be:
```
# Clean up the temporary log file as the contents are captured
try:
    os.remove(log_file)
except OSError as error:
l   log.warning("Unable to remove file {}.".format(log_file))
    log.warning("Output trailer files may contain duplicate information.")
```

Reply:
Nice log messages.